### PR TITLE
feat!: migrate to fastify@v4

### PR DESCRIPTION
**Pull request by metcoder95**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,13 +64,13 @@
     "yargs-parser": "^20.0.0"
   },
   "devDependencies": {
+    "@fastify/autoload": "^4.0.0",
```
Comment (mcollina) [inline package.json:67]: `@fastify/autoload` should be bumped to v5.0.0

File: package.json [hunk 2]
```diff
@@ -64,13 +64,13 @@
...
+    "@fastify/sensible": "^4.0.0",
```
Comment (mcollina) [inline package.json:68]: `@fastify/autoload` should be bumped to v5.0.0

**Review by mcollina — COMMENTED** [review]
The other changes looks good!

File: package.json (hunk 1 above)
Comment (metcoder95) [reply package.json]: Sure! Done [a294e1c](https://github.com/fastify/fastify-cli/pull/490/commits/a294e1ca4e9f836ccda168ac197aa816693891e3)

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**